src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/modelchecking/ModelCheckingStrategy.kt

```diff
@@ -99,8 +99,10 @@ internal class ModelCheckingStrategy(
             ExecutionPart.INIT -> 0
             ExecutionPart.PARALLEL -> currentInterleaving.chooseThread(0)
             ExecutionPart.POST -> 0
+            ExecutionPart.VALIDATION -> 0
         }
         loopDetector.beforePart(nextThread)
+        traceCollector?.passCodeLocation(SectionDelimiterTracePoint(part))
```

Comment:
Why do you need this?

Reply:
To insert sections delimiter trace point. It's convenient to use them lately during report creation. See `SectionDelimiterTracePoint` documentation.